tensorbay/utility/attr.py

```diff
@@ -311,6 +312,14 @@ def _get_operators(annotation: Any) -> Tuple[_Callable, _Callable]:
     )
 
 
+def _get_origin(annotation: Any) -> Any:
+    module = getattr(annotation, "__module__", None)
+    if module in _BUILTINS and version_info < (3, 7):
```

Comment:
we can know the python version at the start of the program, it is redundant to check python version every-time.
find a way to only check python version once.